components/hab/src/cli.rs

```diff
@@ -245,7 +245,9 @@ pub fn get() -> App<'static, 'static> {
                     "Use a specific Depot URL (ex: http://depot.example.com/v1/depot)")
                 (@arg AUTH_TOKEN: -z --auth +takes_value "Authentication token for the Depot")
                 (@arg CHANNEL: --channel -c +takes_value
-                    "Upload to the specified release channel (default: unstable)")
+                    "Additional release channel to upload package to. \
```

Comment:
I thought about this a bit and I think this is the right move for *this* command. I was in the middle of making channel required instead of optional for all of the API points in the client (hab, supervisor, etc) but I think that's just appropriate for read operation. For write I think this is completely correct to make it optional. I also like that we're now telling somebody in the client that we automatically place in unstable regardless. Good work man